### ISSUE TITLE
Implement .csv output file [develop in csv-output branch]

### ISSUE THREAD

**Issue by hdrie** [hdrie/xp-benchmark-tool#5]
Don't start working on this until basic functionality is achieved:
- Log,txt parsing
- Writing to .txt output file

**Comment by hdrie** [comment]
Maybe develop this on a separate branch?
Results strings will need to be formatted much differently.

**Comment by hdrie** [comment]
Initial implementation in [9556c47](https://github.com/hdrie/xp-benchmark-tool/commit/9556c472df9ca216160e511a6802c8ff94132c13)
Needs testing.

**Comment by hdrie** [comment]
Tested and merged into main
